resdk/resolwe.py

```diff
@@ -74,8 +74,10 @@ class Resolwe(ResolweUtilsMixin):
     def __init__(self, username=None, password=None, url=None):
         """Initialize attributes."""
         if url is None:
-            # Try to get URL from environmental variable, otherwise fallback to default.
-            url = os.environ.get('RESOLWE_API_HOST', DEFAULT_URL)
+            # Try to get URL from environmental variables, otherwise fallback to default.
+            url = os.environ.get('RESOLWE_HOST_URL', DEFAULT_URL)
+            # TODO: Remove this
+            url = os.environ.get('RESOLWE_API_HOST', url)
```

Comment:
I would wrap this in:
```
if 'RESOLWE_HOST_URL' not in os.environ:
    url = os.environ.get('RESOLWE_API_HOST', url)
```
so that `RESOLWE_HOST_URL` takes precedence over `RESOLWE_API_HOST`.